src/py/flwr/client/app.py

```diff
@@ -340,13 +383,50 @@ def _load_client_app() -> ClientApp:
     # Initialize connection context manager
     connection, address = _init_connection(transport, server_address)
 
+    retry_invoker = RetryInvoker(
+        exponential,
+        RequestsConnectionError if transport == "rest" else RpcError,
```

Comment:
With the proposal above, we can change this to
```suggestion
        connection_error_type,
```